PanoptesNetClient/PanoptesNetClientTests/RequestTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using NUnit.Framework;
+using PanoptesNetClient;
+using System.Collections.Specialized;
+
+namespace PanoptesNetClientTests_NUnit
+{
+    [TestFixture]
+    public class RequestTests
+    {
+        // <summary>
+        // Test ById correctly adds an attribute and builds a relevant endpoint
+        // </summary>
+        [Test]
+        public void ById()
+        {
+            IRequest request = new Request("projects");
```

Comment:
Yeah, actually looking at this here, I really think it would be a good idea to make specific classes for different `IRequest`s. Then this line would look something like:

`IRequest request = new ProjectRequest()`

This way you enforce what endpoints can be reached and your code should be safer